src/components/Table/components/TableHeader.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import { Input } from 'antd';
+import './TableHeader.scss';
+import searchIcon from '../../../images/svg/searchIcon.svg';
+
+const TableHeader = ({ totalCount, searchValue, handleSearch, handleSearchValueChange }) => {
+  const [input, setInput] = useState('');
+
+  useEffect(() => {
+    setInput(searchValue);
+  }, [searchValue]);
+
+  const handleInputChange = (event) => {
+    const value = event.target.value;
+    setInput(value);
+    handleSearchValueChange(value);
+  };
+
+  const handleSearchSubmit = (event) => {
+    event.preventDefault();
+    handleSearch(searchValue);
+  };
+
+  return (
+    <div className="table-header">
+      <div className="header-content">
+        <h2 className="header-title">All Projects</h2>
```

Comment:
Because we need to reuse this we can not use All Projects here. we need to get this title from props. And then it will be dynamic. If we go to employees page we want to se all employees instead of project. Am I right